notebooks/Example_LLH_setup_fixed_dirs.ipynb

```diff
@@ -787,76 +897,599 @@
+    "t1 = t0 + 2.048\n",
+    "sig_llh_obj.set_time(t0, t1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The minimization process is run using the function .minimize() which returns three objects. The first, **pars**, contains the set of parameters which minimized the likelihood. The second parameter, **nllh**, is the negative log likelihood associated with the parameters, **pars**. The last, **res**, is a return object of the Scipy minimizer object. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#%%time\n",
+    "pars, nllh, res = sig_miner.minimize()\n",
+    "\n",
+    "print(res)\n",
+    "print(nllh)\n",
+    "print(pars)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The negative log likelihood is calculated for the background-only model so that it can be compared to the compound (signal+background) model. Setting the parameter 'Signal_A' to 1e-10 effectively cancels out the contribution from the signal model. Since none of the parameters are being varied, the log likelihood only needs to be calculated once using the .get_logprob() function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "# Evaluating log likelihood for background only model. This is accomplished by setting Signal_A to effectively zero. \n",
+    "pars_['Signal_A'] = 1e-10\n",
+    "bkg_nllh = -sig_llh_obj.get_logprob(pars_)\n",
+    "print (bkg_nllh)\n",
+    "print (sqrtTS)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The likelihood ratio test statistic given by equation 13 (section 6. GRB Search):\n",
+    "\n",
+    "$$\\lambda  = -2[LLH(\\Theta_{bkg}) - LLH(\\Theta_{sig}, \\Theta_{bkg})]$$\n",
+    "\n",
+    "It is used to compare the compare the background+signal model to the background-only model. This statistic is used to quantify the preference for the signal+background model compared to the background-only model. \n",
+    "\n",
+    "In the following block, the square root of the likelihood ratio test statistic is calculated using the negative log likelihood of the background model and the minimized background+signal model. \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sqrtTS = np.sqrt(2.*(bkg_nllh - nllh[0]))\n",
+    "print (sqrtTS)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The value of **sqrtTS** indicates a strong preference for the signal+background model. As a result, we have a strong case for having detected a GRB. By comparison, a value of 8.0 is the threshold for confident detection of a GRB. At a threshold rate, the false alarm rate (FAR) is less than $10^{-4}s^{-1}$ which is much less than 1 per day. "
```

Comment:
"$10^{-4}s^{-1}$ which is much less than 1 per day. "" this is not true!
1/day is 1/86400 s == 1.1e-5 Hz.